app/imports/api/methods/commentsMethods.js

```diff
@@ -71,3 +71,24 @@ export const deleteComment = new ValidatedMethod({
     }
   } 
 });
+
+// This is not a method. IMO since this should to be called only in the server it could be a normal function
+export const removeAllUserComments = (userId) => {
+
+  // Get all comments by the givven user
+  let userComments = Comments.find({
+    userId
+  }, {
+    fields: {
+      applicationId: 1
+    },
+  }).fetch();
+
+  // Get all application IDs related to the comments by the given user
+  let ratedApplicationsIds = _.map(userComments, comment => comment.applicationId);
```

Comment:
why do you do this here? These are applications with comments not ratings. And you don't use this variable later in the comment?